expression/builtin_encryption_test.go

```diff
@@ -310,8 +309,8 @@ var sha1Tests = []struct {
 	{mysql.DefaultCollationName, 1024, "128351137a9c47206c4507dcf2e6fbeeca3a9079"},
 	{mysql.DefaultCollationName, 123.45, "22f8b438ad7e89300b51d88684f3f0b9fa1d7a32"},
 	{"gbk", 123.45, "22f8b438ad7e89300b51d88684f3f0b9fa1d7a32"},
-	{"gbk", "一二三", "01c1743ce7a7e822454a659f659bad61375ff10c"},
-	{"gbk", "一二三123", "7c9a76465a02c41d377596431ef29418e2f6a72c"},
+	{"gbk", "一二三", "30cda4eed59a2ff592f2881f39d42fed6e10cad8"},
```

Comment:
why the test case is out of a test function. can you please move it into the test function?

Reply:
Resolved, also I found `sha2Tests` only used within `TestSha2Hash` and move in